src/App.js

```diff
@@ -13,7 +13,7 @@ const App = () => {
       <Route path="/" element={<Layout />}>
         <Route path="/">
           <Route path="/" element={<SearchPage />} />
-          <Route path=":name" element={<RepoPage />} />
+          <Route path=":full_name" element={<RepoPage />} />
```

Comment:
if I try to type in .../{user} e.g. /scralfie in your deployed version it doesn't seem to like it